nci/lib/watcher.rb

```diff
@@ -186,12 +186,12 @@ def dch
       @dch
     end
 
-    def bump_version
+    def bump_version(series)
       changelog = Changelog.new(Dir.pwd)
       version = Debian::Version.new(changelog.version)
       version.upstream = newest_version
       version.revision = '0neon' unless version.revision.to_s.empty?
-      @dch = Debian::Changelog.new_version_cmd(version.to_s, distribution: NCI.current_series, message: 'New release')
+      @dch = Debian::Changelog.new_version_cmd(version.to_s, distribution: @series, message: 'New release')
```

Comment:
`series` not `@series`, `@` denotes a different type of variable (scoped to the object)